bootiful-banners-service/src/main/java/com/example/BootifulBannersServiceApplication.java

```diff
@@ -63,6 +64,28 @@ public static void main(String[] args) {
             boolean invert = this.properties.isInvert();
 
             String banner = imageBanner.printBanner(maxWidth, aspectRatio, invert);
+            if(ansiOutput == true) {
```

Comment:
OK. Did it better with
````java
            ResourceBanner resourceBanner = new ResourceBanner(new ByteArrayResource(banner.getBytes()));
            
            ByteArrayOutputStream out = new ByteArrayOutputStream();
      
            resourceBanner.printBanner(env, getClass(), new PrintStream(out));
            banner = out.toString();
```
So I will put this in another PR and close this one.
@dsyer thanks for pointing that out !